src/umpire/interface/c_fortran/typesUmpire.h

```diff
@@ -30,11 +30,11 @@
 extern "C" {
 #endif
 
-struct s_UMP_SHROUD_capsule_data {
+struct s_umpire_SHROUD_capsule_data {
```

Comment:
Should the header of this file change too?

Reply:
I'm not sure I understand what you are saying.

Reply:
The copyright/boilerplate header for this file seems different.